dataplane/src/drivers/dpdk.rs

```diff
@@ -20,8 +20,10 @@ use net::packet::Packet;
 use pipeline::sample_nfs::Passthrough;
 use pipeline::{self, DynPipeline, NetworkFunction};
 
+/*
 #[global_allocator]
 static GLOBAL_ALLOCATOR: RteAllocator = RteAllocator::new_uninitialized();
+ */
```

Comment:
Would be nice to provide some context on why we temporary disable this (instead of just removing it).

Reply:
Maybe just add a comment about that in the code?

Reply:
I have commented the issue in the commit.